app/views/issues/map.html.slim

```diff
@@ -0,0 +1,54 @@
+link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"
```

Comment:
Please avoid displaying protocols in URL
Use href="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css" instead of href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"